packages/offix-client/test/apollo/conflicts/ConflictLink.test.ts

```diff
@@ -0,0 +1,53 @@
+import { ConflictLink } from "../../../src/apollo/conflicts/ConflictLink";
+import { GraphQLError } from "graphql";
+import { VersionedState } from "offix-conflicts-client";
+
+/**
+ * Return any so that we can test private methods
+ */
+function newConflictLink(): any {
+  return new ConflictLink({ conflictProvider: new VersionedState() });
+}
+
+test("get conflict error from generic graphql errors", () => {
+  const link = newConflictLink();
+  const result = link.getConflictData([new GraphQLError("a generic error")]);
+  expect(result).toBeUndefined();
+});
+
+test("get conflict error from graphql error with generic extensions", () => {
+  const link = newConflictLink();
+  const result = link.getConflictData([
+    new GraphQLError(
+      "some error",
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      {}
+    )
+  ]);
+  expect(result).toBeUndefined();
+});
+
+test("get conflict error from conflict error", () => {
+  const link = newConflictLink();
+  const result = link.getConflictData([
+    new GraphQLError("some other errors"),
+    new GraphQLError(
+      "some error",
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      {
+        exception: {
+          conflictInfo: {}
+        }
+      }
+    )
+  ]);
+  expect(result).toEqual({});
```

Comment:
Nice test cases. Could this maybe be some other value that makes it more obvious that it's meant to be the `exception.conflictInfo` property. It is not immediately obvious that's what we're matching. Even just some silly string value or maybe add some key/values so it's not an empty object.

Reply:
I've added the full object